src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { PlaylistContext } from '../Context/PlaylistContext';
+import { spotifySearch } from '../../util/Spotify';
+import './SearchBar.css'
+
+export default function SearchBar(){
+
+    const infoContext = useContext(PlaylistContext);
```

Comment:
Instead of having this use the global context, I would either create a smaller context for the Search component, which has both Search and SearchResults, or I would just keep all the state in the Search component, without Context